upgrade/upgrade.go

```diff
@@ -130,6 +136,27 @@ func CreateUpgradeHandler(
 			return vm, errors.Wrap(err, "failed to migrate authority address")
 		}
 
+		// Remove unsupported messages from GlobalFee bypass list.
+		oldBypassMessages, err := globalFeeKeeper.GetBypassMessages(ctx)
+		if err != nil {
+			return vm, err
+		}
+		err = globalFeeKeeper.BypassMessages.Clear(ctx, nil)
+		if err != nil {
+			return vm, err
+		}
+		for _, bypassMessage := range oldBypassMessages {
+			resolved, err := registry.Resolve(bypassMessage)
+			if err != nil || resolved == nil {
+				continue
+			}
+
+			err = globalFeeKeeper.BypassMessages.Set(ctx, bypassMessage)
+			if err != nil {
+				return vm, err
+			}
+		}
```

Comment:
Looks good! 👏 
Holding off on approving until we migrate the rest of circles changed message types.

Reply:
Check out https://github.com/noble-assets/noble/commit/ed4834eff65b9d028c0918effb4612b47aac0d8d, and let me know if you agree!

Reply:
Love it.
Can confirm this works after doing a mock upgrade.

If we do it this way, there are a few messages I think we should add:

```
"/cosmos.upgrade.v1beta1.MsgCancelUpgrade",
"/cosmos.upgrade.v1beta1.MsgSoftwareUpgrade",

"/ibc.core.channel.v1.MsgTimeoutOnClose",
"/ibc.core.client.v1.MsgRecoverClient"
```

Would you agree?